pallets/ethereum-events/src/lib.rs

```diff
@@ -712,6 +715,9 @@ pub mod pallet {
             let account_id = ensure_signed(origin)?;
             ensure!(&tx_hash != &H256::zero(), Error::<T>::MalformedHash);
 
+            let filter = T::EthereumEventsFilter::get_filter();
+            ensure!(!filter.contains(&event_type), Error::<T>::ErrorAddingEthereumLog);
+
             // TODO [TYPE: weightInfo][PRI: medium]: Return accurate weight
             return Self::add_event(event_type, tx_hash, account_id)
```

Comment:
I think the check should be done in this function to ensure all entry points for events are checked:
- add_validator_log
- add_lift_log
- add_ethereum_log